### PR TITLE
CephFS-HA Deployment-CephFS T-2 83575000

### PR DESCRIPTION
**Pull request by hkadam134**
Test Case : https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83575000
Logs : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-21WHZW/

# Description

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/cephfs_utilsV1.py [hunk 1]
```diff
@@ -2916,3 +2915,13 @@ def generate_fio_config(ioengine, mount_dir, workload, size, iodepth, numjobs):
             fio_filenames.append(filename)
         log.info("Generated all the configs")
         return fio_filenames
+
+    @retry(CommandFailed, tries=3, delay=60)
+    def validate_services(self, client, service_name):
+        out, rc = client.exec_command(
```
Comment (AmarnatReddy) [inline tests/cephfs/cephfs_utilsV1.py:2921]: Can you Add Doc string to this function

File: tests/cephfs/cephfs_utilsV1.py [hunk 2]
```diff
@@ -2916,3 +2915,13 @@ def generate_fio_config(ioengine, mount_dir, workload, size, iodepth, numjobs):
             fio_filenames.append(filename)
         log.info("Generated all the configs")
         return fio_filenames
+
+    @retry(CommandFailed, tries=3, delay=60)
```
Comment (AmarnatReddy) [inline tests/cephfs/cephfs_utilsV1.py:2919]: Should we increate the tries or delay for baremetal

Comment (hkadam134) [reply tests/cephfs/cephfs_utilsV1.py:2919]: While testing, the services were up in 1st attempt only.
If we hit any issues later, we can increase the delays and retry count.

**Review by AmarnatReddy — COMMENTED** [review]
Lgtm

**Review by AmarnatReddy — APPROVED** [review]

**Pull request merged**